tests/common/dualtor/nic_simulator_control.py

```diff
@@ -297,7 +297,7 @@ def _set_drop_active_active(interface_names, portids, directions):
             "Set drop recover on port %s, mux server %s, portid %s",
             interface_name, nic_address, portid,
```

Comment:
Minor: if you `_call_set_drop_nic_simulator` in one go, why do you need to log one-by-one then?

Reply:
This is because `_set_drop_active_active` supports set drop for either the upper ToR or the lower ToR via `portid`, so `portid` in the `portids` list might be not the same.